src/client/components/icons/Cross.tsx

```diff
@@ -2,16 +2,15 @@ import React from 'react'
 
 export const Cross = () => (
```

Comment:
nitpick: while we are at it you could consider wrapping these in the `IconRoot` component to standardise how we use icons.

Reply:
Fixed that now on the three icons I used :)